Doc/faq/general.rst

```diff
@@ -133,7 +133,10 @@ Python versions are numbered "A.B.C" or "A.B":
   changes.
 * *C* is the micro version number -- it is incremented for each bugfix release.
 
-See :pep:`6` for more information about bugfix releases.
+See `Status of Python versions section of the Python's Developer Guide
+<https://devguide.python.org/versions/>`__ for more information about
+bugfix releases. Also, consult :pep:`387` to learn more about Python's
+Backward Compatibilty policy.
```

Comment:
Let's point to https://devguide.python.org/developer-workflow/development-cycle/, that's where [PEP 6](https://peps.python.org/pep-0006/) points.

Maybe something like this?

```suggestion
See the `Developer's Guide
<https://devguide.python.org/developer-workflow/development-cycle/>`__ for more information about
the development cycle, and :pep:`387` to learn more about Python's
backward compatibility policy.
```

Reply:
Sounds good! Accepted the suggestion.